src/commands/eval.js

```diff
@@ -59,12 +59,14 @@ function performQuery(client, fqlQuery, outputFile, outputFormat) {
       return writeFormattedOutput(outputFile, response, outputFormat)
     })
     .catch(function (error) {
-      console.log(
+      console.error(
         util.inspect(JSON.parse(error.faunaError.requestResult.responseRaw), {
           depth: null,
           compact: false,
         })
       )
+    
+      process.exit(1)
```

Comment:
I would rather have something on the lines of:
```suggestion
      console.error(JSON.stringify(JSON.parse(error.faunaError.requestResult.responseRaw), null, 2))
      errorOut(new Error(`The query #${err.queryNumber} has failed`))
```

...but it's unclear to me why it's important to keep the consistency added in #118.

I would prefer having the ability to parse the errors than having it looking fancier.

Reply:
Hey @zvictor we're going to punt on making any calls on emitting parseable errors from the shell as we'd want to make that behavior consistent across the share, and think carefully about making error formats part of the shell's contract.

I have pushed a tweak to this section however, that leverages `errorOut` and maintains the formating that was applied in #118 .

There's now a test to confirm the exit code as well.